packages/app/src/specs/generators/EmptyGenerator.vue

```diff
@@ -93,7 +93,7 @@
             query: {
               file: result.file.relative?.replace(/\\/g, '/')
             },
-            params: props.type === 'component'
+            params: props.type === 'component' || 'componentEmpty'
```

Comment:
Is the `componentEmpty` key used for both component and e2e empty spec generation? This creates an issue where the banner is shown for empty e2e specs in the runner. This banner should only be shown for component generated specs since it links to the `mount` documentation